python/rmm/rmm_cfg.py

```diff
@@ -15,8 +15,8 @@
 """Configuration module for librmm, the RAPIDS Memory Manager python interface.
 
    Usage example:
-    from librmm_cffi import librmm_config as rmm_cfg
-    rmm_cf.use_pool_allocator = True
+    from rmm import rmm_cfg
```

Comment:
why rename `config` to `cfg`? Seems like config is just a little bit more clear and nicer looking